Linux AMD64 cannot use _Ctype_longlong(flags) (value of type _Ctype_longlong) as _Ctype_long value in variable declaration

I can't build this on Linux AMD64 because of the following error:

```
../../../go/pkg/mod/pipelined.dev/audio/vst2@v0.10.1-0.20230718065422-be1242fca4ab/plugin.go:141:97: cannot use _Ctype_longlong(flags) (value of type _Ctype_longlong) as _Ctype_long value in variable declaration
```

Changing the given line to

```
return (*TimeInfo)(unsafe.Pointer(uintptr(C.callbackHost(h.callback, cp, C.int(HostGetTime), 0, C.long(flags), nil, 0))))
```

seems to fix the issue and it builds successfully.

--- a/plugin.go
+++ b/plugin.go
@@ -157,7 +157,7 @@ func (h callbackHandler) host(cp *C.CPlugin) Host {
 			return ProcessLevel(C.callbackHost(h.callback, cp, C.int(HostGetCurrentProcessLevel), 0, 0, nil, 0))
 		},
 		GetTimeInfo: func(flags TimeInfoFlag) *TimeInfo {
-			return (*TimeInfo)(unsafe.Pointer(uintptr(C.callbackHost(h.callback, cp, C.int(HostGetTime), 0, C.longlong(flags), nil, 0))))
+			return (*TimeInfo)(unsafe.Pointer(uintptr(C.callbackHost(h.callback, cp, C.int(HostGetTime), 0, C.int64_t(flags), nil, 0))))
 		},
 	}
 }

